turbopack/crates/turbopack-ecmascript/src/minify.rs

```diff
@@ -31,7 +31,18 @@ use turbopack_core::{
 use crate::parse::generate_js_source_map;
 
 #[instrument(level = Level::INFO, skip_all)]
-pub fn minify(code: &Code, source_maps: bool, mangle: Option<MangleType>) -> Result<Code> {
+pub async fn minify(code: Code, source_maps: bool, mangle: Option<MangleType>) -> Result<Code> {
+    let (tx, rx) = tokio::sync::oneshot::channel();
```

Comment:
@lukesandberg I think this is one cause of more allocation, but I think the real cause is that we unblock `tokio` for obvious reasons. 
I updated the PR with the link to the numbers, and I think the numbers can justify this RSS increase.

Reply:
yeah unblocking tokio seems like the most likely culprit.  i cannot imagine a oneshot chanell being more than a few dozen bytes in reality. (your closure would be bigger i think).

totally agree that this is justified. it is faster and you are doing a tokio best practice